notebook/converted/benchmark_results.py

```diff
@@ -1450,8 +1450,66 @@ def return_unlabeled():
 plot
 
 
-# In[ ]:
+# ## Tissue Split
+
+# In[2]:
+
+
+in_files = glob.glob('../../results/tissue-split.*.tsv')
+print(in_files[:5])
+
+
+# In[3]:
+
 
+tissue_metrics = pd.DataFrame()
 
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('biobert.')[-1]
+    model_info = model_info.split('.')
+    model_info = model_info[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+            
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
 
+tissue_metrics
+
+
+# In[4]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='is_pretrained')) 
+plot += geom_smooth()
```

Comment:
Do you have any idea why logistic regression seems to do better here than the NN models?

Reply:
I have no idea. I think there may be a bug in the implementation causing the accuracies to be uniformly low (I'd expect a non-pretrained model to have ~40% accuracy). As for why LR does better, I don't know

Reply:
Yeah, the NN accuracies being so low across the board is a bit surprising. Something to investigate in the future, maybe.